packages/server/modules/automate/migrations/20241203212110_cascade_delete_automations.ts

```diff
@@ -0,0 +1,27 @@
+import { Knex } from 'knex'
+
+export async function up(knex: Knex): Promise<void> {
+  await knex.schema.alterTable('automation_revision_functions', (table) => {

```

Comment:
make sure u try these out locally (both up & down), dropping and re-creating a foreign key within the same alterTable() scope could be problematic

Reply:
can confirm it appears to behave as expected 👍